### ISSUE TITLE
PHP 8.0 Support

### ISSUE THREAD

**Issue by wouter-toppy** [sebastianbergmann/comparator#93]
Is There PHP 8.0 support? 

I want to update to 4.0 but that tags requires 7.3

**Comment by sebastianbergmann** [comment]
I am sorry, but I do not understand what you are refering to. https://github.com/sebastianbergmann/comparator/blob/4.0.6/composer.json#L27 clearly shows that this is supported on PHP 8.

**Comment by wouter-toppy** [comment]
Oh Wow, I'm just such a idiot. 

I've got this error: 
```
  Problem 1
    - sebastian/comparator[4.0.0, ..., 4.0.2] require php ^7.3 -> your php version (8.0.3) does not satisfy that requirement.
    - sebastian/comparator[4.0.3, ..., 4.0.6] require sebastian/diff ^4.0 -> found sebastian/diff[4.0.0, ..., 4.0.4] but the package is fixed to 3.0.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires sebastian/comparator ^4.0 -> satisfiable by sebastian/comparator[4.0.0, ..., 4.0.6].
```

But i didnt notice that it wasn't using the latest tag. 

My bad,  I had to manually update all your packages to `4.0.x` 

Sorry for the incorrect issue.
